classes/03class/exercises/c03-docker06/danielvca/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# FROM alpine:3.12
+# ENTRYPOINT [ "echo", "Hello" ]
+ARG REPO=mcr.microsoft.com/dotnet/core/runtime-deps
+FROM $REPO:3.1-alpine3.12
```

Comment:
you need to let MG go...
```
(λ master) $ docker images|grep alpine
mcr.microsoft.com/dotnet/core/runtime-deps                      3.1-alpine3.12        dfefe13bf579        3 days ago          9.86MB
alpine                                                          latest                a24bb4013296        7 weeks ago         5.57MB
```
you almost doubled the size of an image that runs an `echo` command to use a MS image.. :rofl: